contracts/attrs/src/contract.rs

```diff
@@ -164,423 +176,415 @@ fn try_update_label(
-                        ..
-                    } => {
-                        assert_eq!(name, "label.contract.pb");
-                        assert_eq!(
-                            from_binary::<Label>(original_value).unwrap(),
-                            Label {
-                                text: "original_text".to_string()
-                            }
-                        );
-                        assert_eq!(original_value_type, &AttributeValueType::Json);
-                        assert_eq!(
-                            from_binary::<Label>(update_value).unwrap(),
-                            Label {
-                                text: "update_text".to_string()
-                            }
-                        );
-                        assert_eq!(update_value_type, &AttributeValueType::Json);
-                    }
-                    _ => panic!("unexpected attribute params"),
-                },
-                _ => panic!("unexpected provenance params"),
-            },
-            _ => panic!("unexpected cosmos message"),
-        }
-    }
-
-    #[test]
-    fn query_get_label_name() {
-        // Create default provenance mocks.
-        let mut deps = mock_dependencies(&[]);
-
-        // Init so we have state
-        let _ = instantiate(
-            deps.as_mut(),
-            mock_env(),
-            mock_info("sender", &[]),
-            InitMsg {
-                name: "contract.pb".into(),
-            },
-        )
-        .unwrap(); // Panics on error
-
-        // Call the smart contract query function.
-        let bin = query(deps.as_ref(), mock_env(), QueryMsg::GetLabelName {}).unwrap();
-
-        // Ensure that we got the expected response
-        let rep: LabelNameResponse = from_binary(&bin).unwrap();
-        assert_eq!(
-            rep,
-            LabelNameResponse {
-                name: "label.contract.pb".into()
-            }
-        );
-    }
-
-    #[test]
-    fn query_get_labels() {
-        // Create custom provenance mocks.
-        let mut deps = mock_dependencies(&[]);
-        deps.querier.with_attributes(
-            MOCK_CONTRACT_ADDR,
-            &[("label.contract.pb", "{\"text\":\"text\"}", "json")],
-        );
-
-        // Init so we have state
-        let _ = instantiate(
-            deps.as_mut(),
-            mock_env(),
-            mock_info("sender", &[]),
-            InitMsg {
-                name: "contract.pb".into(),
-            },
-        )
-        .unwrap(); // Panics on error
-
-        // Call the smart contract query function.
-        let bin = query(deps.as_ref(), mock_env(), QueryMsg::GetLabels {}).unwrap();
-
-        // Ensure that we got the expected response
-        let rep: LabelsResponse = from_binary(&bin).unwrap();
-        assert_eq!(rep.labels.len(), 1);
-        assert_eq!(
-            rep.labels[0],
-            Label {
-                text: "text".into(),
-            }
-        )
-    }
-}
+//
```

Comment:
All the tests are commented out. Is this a mistake, or are we planning to introduce them at a different stage?

Reply:
most are back now, but will probably catch the last contract or 2 in the next RC